portfolio/src/main/webapp/script.js

```diff
@@ -33,20 +33,29 @@ function displayLocation(name) {
  */
 function hideLocation(name) {
   var [label, img] = getImage(name); 
-  label.style.display = "none"; 
+  label.style.display = 'none';
   img.style.opacity = 1; 
 }
 
 /**
  * Async function to fetch server content and add it to DOM.
  */
 async function getServerContent() {
-  const response = await fetch('/data');
+  var num = "?num=" + document.getElementById('display-num').value;
```

Comment:
perhaps this portion could be put in a helper getServerContent() to simplify this function and instead this function could probably more accurately be named something like displayServerContent().  As Nick pointed out above, it could be useful to think about ways to split up meaty functions like this into helper functions and condense your logic.

Reply:
Got it - will split getServerContent() into two functions!